scripts/upgradeable/transpile.sh

```diff
@@ -36,7 +37,8 @@ npx @openzeppelin/upgrade-safe-transpiler@latest -D \
   -x '!contracts/proxy/beacon/IBeacon.sol' \
   -p 'contracts/**/presets/**/*' \
   -n \
-  -N 'contracts/mocks/**/*'
+  -N 'contracts/mocks/**/*' \
+  -q '@openzeppelin/'
```

Comment:
@Amxx Does this look right to you? We have to do this because the contract source paths already have a `contracts/` prefix, but it's not really that `@openzeppelin/` is the peer dependency. Ideally I would write something like `@openzeppelin/contracts/=contracts/`, like a Foundry remapping, but I don't want to complicate the transpiler more...

Reply:
I used to do `-q '@openzeppelin/contracts/..` for my tests. What you wrote works just fine.

We can change to a remaping / remapings in a further version of the transpiler